application/assets/images/hmrc-crown-logo_black.svg

```diff
@@ -0,0 +1,175 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Is this file going to need the IP address rule from https://github.com/hmrc/hmrc-frontend/pull/14?

Reply:
Good spot, yes it will, I'll add it